lib/LaTeXML/Package/proof.sty.ltxml

```diff
@@ -0,0 +1,82 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  proof.sty                                                          | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+# The premises can be separated by "&" (which is NOT being used for alignment!)
+DefMath('\lx@proof@logical@and', "\x{2003}", meaning => 'and', role => 'ADDOP');
+DefMacro('\lx@proof@split@and{}', sub {
+    my ($gullet, $tokens) = @_;
+    my @clauses = SplitTokens($tokens, T_ALIGN);
+    if (!@clauses) {
+      return; }
+    elsif (scalar(@clauses) == 1) {
+      return Tokens(@{ $clauses[0] }); }
+    else {
+      return I_apply(undef, T_CS('\lx@proof@logical@and'), map { Tokens(@$_); } @clauses); } });
```

Comment:
`I_apply(undef,` is something I have to go and look up, since I forgot what the undef does here.

... grepping over the code base suggests so far we've only used `I_apply({},` is there a reason why it's instead undef here? 

... oh I see, the first argument is meant to be key-value pairs, and undef would skip over that case as well at the moment. I guess I'll just post my musings as a comment.